### DISCUSSION TITLE
0.26 makes alt-a (and more) for making swedish letters behave in a new way

### DISCUSSION THREAD

**Discussion by marlun** [kovidgoyal/kitty#5445] [Q&A] [answered]
In 0.26 I can no longer create åäö because for some reason some fzf command is run. It also can't seem to display the characters, they just become ". If I change back to 0.25.2 everything works again.

**Comment by marlun** [comment]
Now it happened i 0.25.2 as well but only in tmux. In 0.26 it happens outside tmux as well.

**Comment by kovidgoyal** [comment, marked as answer]
Update to 0.26.1 and dont forget to restart both kitty and tmux fully.

**Reply by marlun** [reply]
It works again, thanks!
